src/main/java/org/folio/service/CustomFieldsServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.folio.service;
+
+import java.text.Normalizer;
+
+import io.vertx.core.Future;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.folio.repository.CustomFieldsRepository;
+import org.folio.rest.jaxrs.model.CustomField;
+import org.folio.service.exc.ServiceExceptions;
+
+@Component
+public class CustomFieldsServiceImpl implements CustomFieldsService {
+
+  @Autowired
+  private CustomFieldsRepository repository;
+
+  @Override
+  public Future<CustomField> save(CustomField customField, String tenantId) {
+    final String customFieldId = unAccentName(customField.getName());
```

Comment:
It might be good to add some logging around the name and id that we are deriving out of the name for a custom field

Reply:
@sduvvuri-ebsco added logging on repositories level as we do for other modules. Additional logging will be added in #7